libcxx/test/libcxx/stacktrace/only_uses_allocator.pass.cpp

```diff
@@ -0,0 +1,112 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++23
+// UNSUPPORTED: asan, msan, tsan, hwasan, sanitizer-new-delete
+// ADDITIONAL_COMPILE_FLAGS: -O0 -g
+
+#include <cassert>
+#include <cstddef>
+#include <cstdlib>
+#include <memory>
+#include <stacktrace>
+
+/*
```

Comment:
Please don't use C-style comments. Also, please move the comment above the includes.